src/components/auth/SignUp.js

```diff
@@ -274,14 +271,14 @@ class SignUp extends React.Component {
                   });
                 }}
               >
-                <MenuItem value='mentor'>Mentor</MenuItem>
-                <MenuItem value='teacher'>Teacher</MenuItem>
+                <MenuItem value="mentor">Mentor</MenuItem>
+                <MenuItem value="teacher">Teacher</MenuItem>
               </Select>
             </FormControl>
             <Button
-              variant='contained'
-              size='large'
-              color='primary'
+              variant="contained"
+              size="large"
+              color="primary"
               onClick={this.handleSubmit}
             >
               Sign Up
```

Comment:
gotta love single to double quote changes 😆

Reply:
See, I'm not the only one who does that!